### DISCUSSION TITLE
Does Tailpress plan to support WordPress Full Site Editing?

### DISCUSSION THREAD

**Discussion by krishaamer** [tailpress/tailpress#112] [Q&A]
Hi @jeffreyvr thanks for all your amazing work! I was wondering if Tailpress has supporting Full Site Editing on the roadmap?

Thanks
Kris

**Comment by jeffreyvr** [comment]
Hi, at this point I don't have clear view of what FSE support would mean for a boilerplate theme. Do you have ideas about this @krishaamer?

**Comment by radscheit** [comment]
@krishaamer Why don't you just turn it into one by yourself? It's a question of a couple of minutes? https://chatgpt.com/share/e/967c66fd-6d06-47bb-bc25-bf8b463c98ae

**Reply by ClosetGeek-Git** [reply]
broken chatgpt links aren't impressive
